api/base/api_globals.py

```diff
@@ -0,0 +1,10 @@
+"""Global variables related to Django requests
+
+Made available in a separate file so as to be importable by Flask code, before (or in the absence of) Django
+"""
+import threading
+
+api_globals = threading.local()
```

Comment:
BTW, this is a genuine question--not rhetorical. I'm not sure if there is any benefit to using werkzeug's implementation, but it's worth looking into.